Sources/Logging/Strings/ConfigureStrings.swift

```diff
@@ -196,6 +197,12 @@ extension ConfigureStrings: LogMessage {
         case .sk2_required:
             return "StoreKit 2 must be enabled. You must use `.with(storeKitVersion: .storeKit2)` " +
             "when configuring the SDK."
+
+        case .fail_post_receipt_due_to_invalid_in_app_purchase_key_configured_in_debug:
```

Comment:
I'd call this something that indicates it's SK2, and a bit shorter, since the return string should be enough explanation: `sk2_invalid_inapp_purchase_key`

Reply:
Yeah, the changes James did only addressed the _public_ API changes with observer mode, not the internal ones. I'll go ahead and update this one here to leave it a little better than I found it! 😄

Reply:
Renamed to `record_purchase_requires_purchases_made_by_my_app`!